plugins/services/src/js/containers/tasks/TasksView.js

```diff
@@ -246,36 +236,26 @@ class TasksView extends mixin(SaveStateMixin) {
       rightAlignLastNChildren = 1;
     }
 
-    filteredTasks = filteredTasks.map(GraphQLTaskUtil.mergeData);
+    const mergedTasks = tasks.map(GraphQLTaskUtil.mergeData);
 
     return (
       <div className="flex-container-col flex-grow">
         <FilterHeadline
-          currentLength={filteredTasks.length}
+          currentLength={mergedTasks.length}
           inverseStyle={inverseStyle}
-          isFiltering={filterByStatus !== "all" || searchString !== ""}
-          onReset={this.resetFilter}
           name={"task"}
-          totalLength={tasks.length}
+          totalLength={totalTasks}
+          onReset={() => {
```

Comment:
btw, no need for the block in the arrow function here.

can be `onReset={() => handleExpressionChange({ value: "" })`